python/py-rawpy/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy v0.17.2
+fetch.type          git
+
+python.versions     310
+
+maintainers         openmaintainer @jsalort
+categories-append   graphics
+
+description         Python wrapper for the LibRaw library
+long_description    rawpy is an easy-to-use Python wrapper for the LibRaw library. It also contains some extra \
+                    functionality for finding and repairing hot/dead pixels.
+
+homepage            https://github.com/letmaik/rawpy
```

Comment:
this is set already by the `github` PG so please remove this line

Reply:
OK, will do.